cli/command/stack/kubernetes/loader.go

```diff
@@ -1,24 +1,33 @@
 package kubernetes
 
 import (
+	"github.com/docker/cli/cli/compose/loader"
 	composetypes "github.com/docker/cli/cli/compose/types"
-	apiv1beta1 "github.com/docker/cli/kubernetes/compose/v1beta1"
 	yaml "gopkg.in/yaml.v2"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-// LoadStack loads a stack from a Compose config, with a given name.
-func LoadStack(name string, cfg composetypes.Config) (*apiv1beta1.Stack, error) {
+func loadStack(name string, cfg composetypes.Config) (stack, error) {
 	res, err := yaml.Marshal(cfg)
```

Comment:
we only need to do that in `v1beta1` case right ? We shouldn't even "try" to do it all the time.